app/src/main/java/de/fraunhofer/fokus/OpenMobileNetworkToolkit/Iperf3/Iperf3Fragment.java

```diff
@@ -285,6 +288,24 @@ public void afterTextChanged(Editable s) {
             LinearProgressIndicator.INDETERMINATE_ANIMATION_TYPE_CONTIGUOUS);
         progressIndicator.setVisibility(LinearProgressIndicator.INVISIBLE);
 
+        btnPrefill.setOnClickListener(v -> {
+            // Set URL
+            iperf3EtIp.setText("demo.app.cloudcities.co");
+
+            // Set port
+            int validPortMin = 9200;
+            int validPortMax = 9240;
+            Random rnd = new Random();
+            int randomPort = rnd.nextInt((validPortMax - validPortMin) + 1) + validPortMin;
+            iperf3EtPort.setText(String.valueOf(randomPort));
+
+            // Set duration to 30
+            iperf3EtDuration.setText(String.valueOf(30));
+
+            // tick of checkboxes
+            iperf3BiDir.setChecked(true);
```

Comment:
some holder for this data would have been better but it does the trick for now. leave it like this.